charts/prometheus-pgbouncer-exporter/Chart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v2
+name: prometheus-pgbouncer-exporter
+description: Prometheus exporter for pgbouncer metrics
+home: https://github.com/prometheus-community/helm-charts
+sources:
+  - https://github.com/prometheus-community/pgbouncer_exporter
+
+keywords:
+  - prometheus
+  - pgbouncer
+
+maintainers:
+  - name: Pluggi
+    email: pluggi512@gmail.com
```

Comment:
I think we need more than one maintainer. I (zanhsieh, zanhsieh@gmail.com) could help a little bit.

Reply:
I added you @zanhsieh :)
Thank you